### ISSUE TITLE
Better sorting song by artist

### ISSUE THREAD

**Issue by pyotr71** [quodlibet/quodlibet#4077]
The proposed change
-------------------
Option to omit **_the_** in artist names.

Benefits of this change
-----------------------
Much better sorting of artist.

**It looks like this now:**
1. Cream
2. Metallica
3. The Beatles

**After add option:**
1. The Beatles
2. Cream
3. Metallica

**Comment by declension** [comment]
Yes, this is `artistsort`. There are `sort` versions of all people-like tags, including `~peoplesort` which is used by default in a few views actually.

If you haven't _got_ these tags added, you can:
* add them manually
* Use the _Create Sort Tags_ plugin to do this semi-automatically (but "badly", as it says)
* Use musicbrainz's tagging (try the QL plugin!) to aqcuire these. They have [detailed rules about sort tags](https://musicbrainz.org/doc/Style/Artist/Sort_Name).

**Comment by pyotr71** [comment]
Thanks! It's good idea, but in Cantata music player sorting works without changing / adding tags.
